README.md

```diff
@@ -147,9 +181,43 @@ With the Azure login Action, you can perform an Azure login using [Azure service
   }
   
 ```
+  *  Alternatively,  to create service principal that has access over management group scope, run the below Azure CLI command.
```

Comment:
The  `sp` API with the `sdk-auth` flag will fail even if one of the scopes present is a management group. Should we rephrase this line to express that?

Reply:
updated. Thanks